### ISSUE TITLE
VR keyboard not opening + Suggestion

### ISSUE THREAD

**Issue by Shazuli** [jrbudda/minecrift#309]
Hi!

[Issue]
As the title says, certain clickable text fields cannot open the Steam VR keyboard, it is visible for a short second before closing again. JEI's search filter has this problem, showing the keyboard for a split second before closing it again. It still adds characters from the controller input after selecting it, so if you press the move forward button it writes `W` in the text field, and clears the field when pressing `right-click` for the emulated mouse.

It doesn't show up at all when opening [OpenComputer](https://minecraft.curseforge.com/projects/opencomputers)'s monitors with a keyboard attached. It doesn't write any characters to the screen when pressing the controller buttons either. They do render stuff a bit special for the screens, but it would be nice to be able to write stuff to the screen.

[Suggestion]
Have been playing around with Vivecraft for a few days now. It's fine on singleplayer, but it can become a bit tedious after a while when having to type to other people one button at a time. Feels like I'm typing with only one finger. Would be nice to have a custom keyboard similar to the buttons on the menu, and instead of having to point and click, have it be touch sensitive when reaching out to it. Would be nice to have as an option whenever you want this or the old Steam keyboard.
And lastly; have it not hardcoded, so you can have your own keyboard layout with custom buttons. This should not be a config option but a file anyone can edit and replace (preferably a `json` file). I'm sure if you guys made a default file with all the regular English characters, the community will probably create all kinds of keyboard layouts!

MC: 1.12.2
Vivecraft: 1.12.2-jrbudda-6-final-r3-installer

**Comment by Shazuli** [comment]
It seems like I got the "force keyboard" keybind to work now. It didn't work at all until just recently, but after a restart it resolved itself. But I still stand on the "suggestion" part.
